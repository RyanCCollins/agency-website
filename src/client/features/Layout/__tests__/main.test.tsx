import * as React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Main from '../main';

describe('Layout Main component ', () => {

  it('should create a snapshot', () => {
    const wrapper = shallow(
      <Main />,
    );
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
