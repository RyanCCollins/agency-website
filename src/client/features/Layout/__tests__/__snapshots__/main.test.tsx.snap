// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout Container  should create a snapshot 1`] = `
<main
  className="DvkhP"
/>
`;

exports[`Layout Main component  should create a snapshot 1`] = `
<main
  className="DvkhP"
/>
`;
